aead/src/lib.rs

```diff
@@ -20,7 +20,7 @@
     html_logo_url = "https://raw.githubusercontent.com/RustCrypto/media/8f1a9894/logo.svg",
     html_favicon_url = "https://raw.githubusercontent.com/RustCrypto/media/8f1a9894/logo.svg"
 )]
-#![warn(missing_docs, rust_2018_idioms)]
+#![warn(missing_docs, rust_2021_compatibility)]
```

Comment:
These lints do different things. Please see this section: https://doc.rust-lang.org/edition-guide/editions/advanced-migrations.html#writing-idiomatic-code-in-a-new-edition

> - Edition 2018:
[unused-extern-crates](https://doc.rust-lang.org/rustc/lints/listing/allowed-by-default.html#unused-extern-crates)
[explicit-outlives-requirements](https://doc.rust-lang.org/rustc/lints/listing/allowed-by-default.html#explicit-outlives-requirements)
> - Edition 2021 does not have any idiom lints.

Reply:
Oh, my mistake. Let me reverse these changes.